lib/src/r_tree/r_tree.dart

```diff
@@ -61,10 +62,277 @@ class RTree<E> {
     return _root.search(searchRect, shouldInclude);
   }
 
-  _growTree(Node<E> node1, Node<E> node2) {
-    NonLeafNode<E> newRoot = NonLeafNode<E>(_branchFactor);
-    newRoot.addChild(node1);
-    newRoot.addChild(node2);
+  /// Bulk adds all [items] to the rtree. This implementation draws heavily from
+  /// https://github.com/mourner/rbush and https://github.com/Zverik/dart_rbush.
```

Comment:
Have you looked into consuming https://github.com/Zverik/dart_rbush/ and using it instead?

Reply:
I didn't realize our implementation has faster reads and we're just trying to improve our load/write implementation. My comment makes less sense now.